hansol/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "hansol",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "typeorm": "^0.3.10"
```

npm mysql 패키지가 설치되어 있지 않습니다. 해당 부분 확인 후 다시 수정해 주세요!
```suggestion
  "dependencies": {
    "cors": "^2.8.5",
    "dotenv": "^16.0.3",
    "express": "^4.18.2",
    "morgan": "^1.10.0",
    "typeorm": "^0.3.10"
```